islandora_embed.module

```diff
@@ -180,11 +180,15 @@ function uuid_embed_object() {
 }
 
 function getTranslatedUrl($url) {
+    module_load_include('inc', 'islandora_embed', 'includes/utilities');
     try{
         $url_arr = explode("/embed/uuid/",$url);
         $uuid = $url_arr[count($url_arr)-1];
-        $translatedUrl = $url_arr[0]."/islandora/embed/object/islandora%3A".$uuid;
-        //$translatedUrl = "/islandra/embed/object/islandora%3A".$uuid;
+	$uuid_arr = explode("?", $uuid);
```

Comment:
This is probably an OK way to do the required URL processing, but it'd be nice to have comments or something that made it a little clearer  what was happening here.

Reply:
Done.